docs/build/wasm/from-zero-to-ink-hero/manic-minter/manic-setup.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 2
+---
+
+# ManicMinter Setup
+This is a tutorial on creating a minter contract and a token contract using the ink! smart contract framework. In this tutorial, you will learn how to develop two contracts, use cross contract call and test it with ink! e2e framework.
+
+The minter contract will handle the minting of new fungible tokens, while the token contract will adhere to the PSP22 standard. Our chosen name for the fungible token smart contract is "Oxygen," and the minter contract will be called "ManicMinter."
+
+Once the contracts are created, the ManicMinter contract will become the owner of the Oxygen contract. Only the ManicMinter contract will have the ability to mint Oxygen tokens, and users will acquire these tokens by paying native tokens to the ManicMinter contract at a price determined by its owner.
+
+Let's help Willy to mint some Oxygen tokens through the ManicMinter contract!
+
+## Prerequisites
+Please refer to the [previous section](./manic-minter.md) for the list of prerequisites.
+
+## ManicMinter and Oxygen Smart Contracts
+### Initial Setup
+In a new project folder, execute the following:
+
+```bash
+$ cargo contract new manicminter
+$ cargo contract new oxygen
+```
+Create the root `Cargo.toml` file with the workspace content:
+```toml
+[workspace]
+members = [
+    "oxygen",
+    "manicminter",
+]
+```
+
+### Oxygen Contract Setup
+Using the Openbrush [wizard](https://openbrush.io/), we will create a new ink! smart contract for PSP22. Provide the following information to the wizard:
```

Comment:
It is not too buggy to be used ?

Reply:
removed wizard and added boilerplate PSP22